src/app/client/src/app/modules/learn/components/course-consumption/course-consumption-header/course-consumption-header.component.ts

```diff
@@ -70,14 +71,17 @@ export class CourseConsumptionHeaderComponent implements OnInit, AfterViewInit,
   }
 
   ngOnInit() {
+    this.courseHierarchy['trackable.enabled'] = 'Yes';
```

Comment:
This code has to be removed. Ideally, you get this value in the hierarchy response itself.